clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -310,7 +310,7 @@ public void destroy(String name) throws CachePersistenceException {
     try {
       entity.destroyCache(name);
     } catch (ClusteredTierDestructionException e) {
-      throw new CachePersistenceException("Cannot destroy clustered tier '" + name + "' on " + clusterUri, e);
+      throw new CachePersistenceException(e.getMessage() + " (on " + clusterUri + ")", e);
```

Comment:
Opinionated decision that the cluster information really belongs at the end of the underlying exception message.